packages/adapters/subgraph/src/lib/operations/queries.ts

```diff
@@ -585,6 +586,7 @@ const originTransferQueryFallbackString = (
       destinationDomain_in: [${destinationDomains}]
       ${maxBlockNumber ? `, blockNumber_lte: ${maxBlockNumber}` : ""}
     },
+    first: 1000,
```

Comment:
I think ideally these should be passed in, and paginated at the app layer so the app knows how much to paginate by instead of hardcoding.

Reply:
tried it here https://github.com/connext/monorepo/pull/4497/commits/523adadccac96bfcbab61de81b8a49e7400c0518